src/main/scala/controllers/ButtonAction.scala

```diff
@@ -20,13 +22,18 @@ case object SaveItem extends ButtonAction {
   override val name: String = "_save_item_button"
 }
 
+case object CompleteAndPreview extends ButtonAction {
+  override val name: String = "_preview_and_complete_button"
```

Comment:
I know it's not actually used, but maybe change this to match the name of the case object (i.e. "_complete_and_preview_button"